test/BUILD

```diff
@@ -20,6 +20,7 @@ envoy_cc_test_library(
     deps = [
         "//source/common/common:thread_lib",
         "//source/common/event:libevent_lib",
+        "//source/exe:sigaction_lib",
```

Comment:
Yes, good catch.  We do want the select for Google (the binaries we will run have this capability through a different mechanism).